model/task_lifecycle.go

```diff
@@ -168,6 +168,17 @@ func TryResetTask(taskId, user, origin string, detail *apimodels.TaskEndDetail)
 			grip.Debugln(message, "marking as failed")
 			if detail != nil {
 				updates := StatusChanges{}
+				if t.DisplayOnly {
+					for _, et := range t.ExecutionTasks {
+						execTask, err := task.FindOne(task.ById(et))
```

Comment:
I think it could be more logical to change `et` to `etId` and `execTask` to just `et`.